docs/getting-started/config/database.md

```diff
@@ -68,17 +68,27 @@ That's enough database administration to get started. Type `\q`, followed by
 pressing the enter key. Then again type `exit` and press enter. Next, you need
 to install and configure the client.
 
-### 2. Configuring Backstage `pg` Client
+### Docker Alternative
 
-Go to the root directory of your freshly installed Backstage
-App. Run the following to install the PostgreSQL client into your backend:
+Alternatively you can run Postgres in a Docker container, this is great for local development or getting a Backstage POC up and running quickly, here's how:
 
-```bash title="From your Backstage root directory"
-yarn --cwd packages/backend add pg
+First we need to pull down the container image, we'll use Postgres 17, check out the [Postgres Version Policy](../../overview/versioning-policy.md#postgresql-releases) to learn which versions as supported.
```

Comment:
s/versions as supported/versions are supported/

Reply:
Nice, good catch, fix this up with my latest changes 👍